Dockerfile

```diff
@@ -1,8 +1,9 @@
 FROM debian:jessie
 RUN apt-get update && \
-    apt-get install -y iptables ca-certificates aufs-tools && \
+    apt-get install -y curl iptables ca-certificates aufs-tools && \
     apt-get clean && \
     rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 ADD ./keys /root/keys
 ADD ./scripts /root/scripts
-ADD https://github.com/concourse/concourse/releases/download/v1.0.0/concourse_linux_amd64 /root/concourse
+RUN curl -L https://github.com/concourse/concourse/releases/download/v1.0.0/concourse_linux_amd64 > /root/concourse
```

Comment:
use ADD here as well, it will work with URLs then we don't have to add the curl package.

I had removed the chmod because I was using a local file, but unfortunately github wouldn't check it in due to file size. Adding the chmod makes the image a lot larger, but is necessary.